presqt/api_v1/views/resource.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from presqt.api_v1.helpers.function_router import FunctionRouter
+from presqt.api_v1.serializers.resource import ResourcesSerializer
+
+
+class ResourcesList(APIView):
+    """
+    **Supported HTTP Methods**
+
+    * Get: Retrieve a summary of all resources for the given Target.
+    """
+    required_scopes = ['read']
+
+    def get(self, request, target_name):
+        """
+
+        Parameters
+        ----------
+        target_name : str
+            The string name of the Target resource to retrieve.
+
+        Returns
+        -------
+        A list-like JSON representation of all resources for the given Target and token.
+
+        [
+            {
+                "kind": "container",
+                "kind_name": "folder",
+                "id": "a02d7b96-a4a9-4521-9913-e3cc68f4d9dc",
+                "container": "None"
+            },
+            {
+                "kind": "item",
+                "kind_name": "file",
+                "id": "5b305f1b-0da6-4a1a-9861-3bb159d94c96",
+                "container": "a02d7b96-a4a9-4521-9913-e3cc68f4d9dc"
+            }
+        ]
+
+        Resources
+        ---------
+        400: Bad Request
+
+        {
+            "error": "'presqt-source-token' missing in the request header."
+        }
+
+        404: Not Found
+        {
+            "error": "'bad_target' is not a valid Target name."
+        }
+        """
+        try:
+            token = request.META['HTTP_PRESQT_SOURCE_TOKEN']
```

Comment:
In text you reference this just a `presqt-source-token` without the HTTP and in lowercase.
It seems like this could be even better to make this consistent.

Unless you want the HTTP for a particular reason, maybe just PRESQT_SOURCE_TOKEN?